sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/TableResourcesOperationsTests.cs

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.CosmosDB.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    [TestFixture]
+    public class TableResourcesOperationsTests : CosmosDBManagementClientBase
+    {
+        private string resourceGroupName;
+        private const string databaseAccountName = "db2672";
+        private const string tableName = "tableName2527";
+        private const int sampleThroughput = 700;
+        private const int defaultThroughput = 400;
+        private const int defaultMaxThroughput = 4000;
+        public TableResourcesOperationsTests()
+            : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                InitializeClients();
+                this.resourceGroupName = Recording.GenerateAssetName(CosmosDBTestUtilities.ResourceGroupPrefix);
+                await CosmosDBTestUtilities.TryRegisterResourceGroupAsync(ResourceGroupsOperations,
+                    CosmosDBTestUtilities.Location,
+                    this.resourceGroupName);
+            }
+        }
+
+        [TearDown]
+        public async Task CleanupResourceGroup()
+        {
+            await CleanupResourceGroupsAsync();
+        }
+
+        [TestCase]
+        public async Task TableCRUDTest()
+        {
+            CosmosDBManagementClient cosmosDBMgmtClient = GetCosmosDBManagementClient();
+
+            // prepare a database account
+            List<Location> locationList = new List<Location>();
+            locationList.Add(new Location(id: null, locationName: "WEST US", documentEndpoint: null, provisioningState: null, failoverPriority: null, isZoneRedundant: null));
+            IEnumerable<Location> locations = locationList;
```

Comment:
You don't need to do this "casting here" I think, just pass in locationList to the create call